pacman-helper.sh

```diff
@@ -248,6 +274,24 @@ quick_add () { # <file>...
 			# skip explicit signatures; we copy them automatically
 			continue
 			;;
+		*-i686|*-x86_64|*-aarch64)
+			test remove = "$label" || die "Cannot add $path"
+			arch=${file##*-}
+			file=${file%-$arch}
+			file=${file%-[0-9]*}
+			file=${file%-[0-9]*}
```

Comment:
Would it be better/safer to do
```suggestion
			file=${file%-[0-9]*-[0-9]*}
```
and require that if you give a version, you give it in full (including pkgrel)?

If you can enable `extglob` you could maybe use a pattern like `-*([^-])-*([^-])`?  And match filenames with `-*([^-])-*([^-])-*([^-]).pkg.tar.*([^.])` instead of `-[0-9]*`?  (some of those glob patterns seem to be interpreted by git instead of by bash before calling git, so I don't know if extglob would work in that case)

Reply:
That is a valid concern. For example, `quick_remove msys2-runtime-3.3-x86_64` would clearly like to remove the `msys2-runtime-3.3` package, not the `msys2-runtime` package. I'll apply your change and squash it in.